feature-libs/asm/root/index.ts

```diff
@@ -0,0 +1,2 @@
+export * from './facade/index';
+export * from './model/index';
```

Comment:
I don't think we need this file. Or, we put them in `index.ts`
```
export * from './asm-constants';
export * from './asm-loader.module';
export * from './asm-root.module';
export * from './facade/index';
export * from './feature-name';
export * from './model/index';
export * from './services/index';
```
and `export * from './index';` in `public_api.ts`.